persistence/pg.go

```diff
@@ -81,7 +83,10 @@ func unmarshallDbInvoice(invoice *dbInvoice) *Invoice {
 				Value:           lnwire.MilliSatoshi(invoice.AmountMsat),
 			},
 		},
-		Settled: invoice.Settled,
+		SequenceNum:       invoice.SequenceNum,
+		Settled:           invoice.Settled,
+		SettleRequestedAt: invoice.SettleRequestedAt,
```

Comment:
Hmm an earlier omission?

Reply:
Yep, it seems (I think we didn;t need it)